scripts/common_functions.ps1

```diff
@@ -1,4 +1,4 @@
-﻿function login_azure {
```

Comment:
do not change this line

Reply:
there were some weird charachters from an editor, just removed those